includes/admin/class-sensei-onboarding.php

```diff
@@ -155,4 +163,85 @@ public function add_onboarding_help_tab( $screen ) {
 			]
 		);
 	}
+	/**
+	 * Register REST API route.
+	 */
+	public function register_rest_api() {
+
+		register_rest_route(
+			'sensei/v1',
```

Comment:
I also mentioned this elsewhere, but I think we should use a namespace like `sensei-internal` for this.